include/crm/common/logging_internal.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef LOGGING__INTERNAL_H
+#  define LOGGING__INTERNAL_H
+
+/*!
+ * \internal
+ * \brief Initialize logging for command line tools
+ *
+ * \param[in] name      The name of the program
+ * \param[in] verbosity How verbose to be in logging
+ *
+ * \note \p verbosity is not the same as the logging level (LOG_ERR, etc.).
+ */
+void pcmk__cli_init_logging(const char *name, unsigned int verbosity);
+
```

Comment:
There's an "internal logging utilities" section of include/crm/common/internal.h that could move here (creating logging_internal.h with those could be a separate commit, to make this one cleaner).

Also, this commit message needs the new function name.

Reply:
Which utilities are these?  I only see the deprecated block at the bottom.